exonum-java-binding/core/rust/src/runtime/config.rs

```diff
@@ -55,15 +55,6 @@ pub struct RuntimeConfig {
     /// A port of the HTTP server for Java services.
     /// Must be distinct from the ports used by Exonum.
     pub port: i32,
-}
-
-/// Internal EJB configuration.
-///
-/// Not visible by user, used internally while initializing runtime.
-#[doc(hidden)] // For testing purposes only.
-pub struct InternalConfig {
-    /// EJB system classpath.
-    pub system_class_path: String,
     /// EJB library path.
     pub system_lib_path: Option<String>,
```

Comment:
It either has the wrong name: override_system_lib_path/custom_system_lib_path
or it must not be an Option.

I am a little confused by which code shall set up the library path to the
default (application) value of "lib/native" — it used to happen in ServiceFactoryAdapter,
currently ServiceRuntime does it itself. Is it the right responsibility for ServiceRuntime?

As it looks rather confusing, I'd improve the thing.